plugins/modules/ec2_placement_group.py

```diff
@@ -43,6 +43,11 @@
     default: cluster
     choices: [ 'cluster', 'spread', 'partition' ]
     type: str
+  tags:
+    description:
+      - A dict of key value pairs to associate with the placement group
+    type: dict
+    version_added: 9.0.0
```

Comment:
```suggestion
    version_added: 9.0.0
```

I think we can release it with 8.1.0

Reply:
Changed to version_added back to 8.1.0